src/sass/partials/_sidebar.scss

```diff
@@ -1,3 +1,5 @@
+$dark-nav-border: #4f5255;
```

Comment:
Can you use PF var `--pf-global--BackgroundColor--dark-400` instead of direct value?

Reply:
Yeah!